spec/jobs/form526_confirmation_email_job_spec.rb

```diff
@@ -112,6 +142,10 @@
           Form526ConfirmationEmailJob.perform_async(personalization_parameters)
         end.to change(Form526ConfirmationEmailJob.jobs, :size).by(1)
       end
+
+      it 'feature toggle' do
+        # if Flipper.enabled?(:vanotify_service_enhancement)
```

Comment:
Looks like this is incomplete.

Reply:
Thanks for noticing it, we added new unit tests to the top of the file and forgot to remove this one.